File: 7kyu/(7 kyu) Build a square/(7 kyu) Build a square.cpp
#include <iostream>
#include <string>

using namespace std;

std::string generate_shape(int n)
{
    std::string res;
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
            res += "+";
        if (i < n - 1)
            res += "\n";
    }
    return res;
}

File: 7kyu/(7 kyu) Build a square/(7 kyu) Build a square.java
import java.util.*;

public class Kata {
    public static final String generateShape(int n) {
        List<String> res = new ArrayList<>();
        for (int i = 0; i < n; i++) {
          res.add("+".repeat(n));
        }
        return String.join("\n", res);
    }
}
